src/main/webapp/view.js

```diff
@@ -13,16 +13,19 @@
 // limitations under the License.
 
 const PARAM_ID = 'id';
+const PARAM_VIDEO_ID = 'video-id';
 
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
+window.VIDEO_ID = getVideoId();
 
 initialize();
 
-/** Initializes different portions of project */
+/**
+ * Initializes the video section of the lecture view page.
```

Comment:
what about:
Initializes the video player for the lecture view page.

"video section" is a bit vauge.